doc/LLGO Migration for C C++ Third-Party Libraries.md

```diff
@@ -0,0 +1,461 @@
+# LLGO Migration for C/C++ Third-Party Libraries
+
+# LLGO Migration for C Third-Party Libraries
+
+## Installing Third-Party Libraries
+
+### Using Package Manager to Download
+
+```bash
+brew install inih
+```
+
+### Compiling from Source
```

Comment:
It depends. There is no need to describe how to install from source code, just prompt to prioritize binary installation unless the package you want to support is not supported.

Reply:
Some people need to compile and install from source code, such as those who migrate godot